Example/iOS/Views/AnimatedButtonRow.swift

```diff
@@ -0,0 +1,111 @@
+// Created by Cal Stephens on 1/4/22.
+// Copyright © 2022 Airbnb Inc. All rights reserved.
+
+import Epoxy
+import Lottie
+import UIKit
+
+// MARK: - AnimatedButtonRow
+
+final class AnimatedButtonRow: UIView, EpoxyableView {
+
+  // MARK: Lifecycle
+
+  init() {
+    super.init(frame: .zero)
+    translatesAutoresizingMaskIntoConstraints = false
+    layoutMargins = UIEdgeInsets(top: 16, left: 24, bottom: 16, right: 24)
+    group.install(in: self)
+    group.constrainToMarginsWithHighPriorityBottom()
+    backgroundColor = .systemBackground
+  }
+
+  required init?(coder _: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+
+  // MARK: Internal
+
+  struct Content: Equatable {
+    var animationName: String
+    var title: String
+    var playRanges: [(fromMarker: String, toMarker: String, event: UIControl.Event)]
+
+    static func == (lhs: Content, rhs: Content) -> Bool {
+      guard
+        lhs.animationName == rhs.animationName,
+        lhs.title == rhs.title
+      else { return false }
+
+      for (lhsRanges, rhsRanges) in zip(lhs.playRanges, rhs.playRanges) {
+        if lhsRanges != rhsRanges {
+          return false
+        }
+      }
+
+      return true
+    }
```

Comment:
nit: making `playRanges` a nested `Equatable` struct would probably be shorter than this implementation

Reply:
good idea